hack/community/readme.md

```diff
@@ -1,21 +1,50 @@
 # Cut a new community release
-The generate.sh script is called in the pre step to set up for the 
-aws-e2e-operator-test. The script copies over the WMCO community 
-bundle to the artifact directory while also replacing csv fields such as the 
-name, description, and date. Once generated into the artifact directory, the 
-bundle is used in the windows-e2e-operator test. After it passes, the bundle can
-be used to release a new community WMCO.
+CI contains a COMMUNITY environment variable that defaults to false, and is only
+set to true when a community-4.x branch PR is opened. When true, CI is 
+signaled to copy the repo's bundle into the test artifact directory, edit the bundle, and then use the revised 
+bundle in 
+the PR's 
+aws-e2e-operator test. After the test passes, the bundle can be copied from the PR's artifact 
+directory and used to release a new community WMCO.
 
-### After the test passes
+### Benefit
+Previously, a manual testing process was done to ensure that the changes to the
+Red Hat operators WMCO manifests were brought in while maintaining the
+differences needed by the community operator. This new process automates the
+testing portion of a community WMCO release by reducing the manual intervention
+done in the bundle and testing the bundle directly in CI.
+
+### Process
 - Fork and clone [community-operators-prod](https://github.com/redhat-openshift-ecosystem/community-operators-prod)
 repo.
-- Copy the community bundle from the ARTIFACT_DIR.
+- Copy the manifests/ and metadata/ directories from the aws-e2e-operator test 
```

Comment:
Given this is an ordered process, I suggest including numbering bullet points. For example:
1. Copy the manifests ...
2. Paste the bundle contents ...
3. ...
N. The last step